app/assets/stylesheets/components/_nav_nav-tabs.scss

```diff
@@ -0,0 +1,19 @@
+.nav-link.active {
+  background-color: #14E78E !important;
```

Comment:
```suggestion
  background-color: $primary !important;
```

Reply:
Perhaps we can change the colour to use our primary colour so that if the primary colour ever needs to be changed, we only need to change it on our assets/stylesheets/config/bootsdtrap_variable file?